### PR TITLE
Release/0.0.4

### PR DESCRIPTION
**Pull request by DavidRedlich**
included Serilaization for TIntFloatMap

### CODE REVIEW

File: src/test/java/com/targomo/jackson/datatype/trove/deser/TIntFloatMapDeserializerTest.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    @Test
+    public void testSimple() throws IOException
+    {
+
+        TIntFloatMap map = reader.forType(TIntFloatMap.class)
+                .readValue("{\"8\" : 1.0, \"1\" : 2.5}");
+        assertEquals(2, map.size());
+        assertThat(map.get(8)).isEqualTo(1.0f);
+        assertThat(map.get(1)).isEqualTo(2.5f);
+        assertThat(map.get(3)).isEqualTo(Integer.MAX_VALUE);
```
Comment (gerbsen) [inline src/test/java/com/targomo/jackson/datatype/trove/deser/TIntFloatMapDeserializerTest.java]: why would this return int.max_value?

File: src/test/java/com/targomo/jackson/datatype/trove/deser/TIntFloatMapDeserializerTest.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+        TIntFloatMap map = new TIntFloatHashMap();
+        map.put(13, 1.0f);
+        map.put(15, 2.5f);
+        map.put(17, -3.886757545f);
+
+        ObjectMapper om = new ObjectMapper();
+        om.registerModule(new TroveModule(Integer.MAX_VALUE))
+                .registerModule(new JodaModule());
+        System.out.println( om.writerWithDefaultPrettyPrinter().writeValueAsString(map) );
```
Comment (gerbsen) [inline src/test/java/com/targomo/jackson/datatype/trove/deser/TIntFloatMapDeserializerTest.java]: syso's in tests?

File: src/test/java/com/targomo/jackson/datatype/trove/ser/TIntFloatMapSerializerTest.java [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+import static org.junit.Assert.assertEquals;
+
+public class TIntFloatMapSerializerTest extends BaseTroveSerializerTest {
+
+    @Test
+    public void testSimple() throws IOException
+    {
+        TIntFloatMap map = new TIntFloatHashMap();
+        map.put(16, 1.5f);
```
Comment (gerbsen) [inline src/test/java/com/targomo/jackson/datatype/trove/ser/TIntFloatMapSerializerTest.java:17]: maybe also add on for the edge case: empty map?

File: src/test/java/com/targomo/jackson/datatype/trove/deser/TIntFloatMapDeserializerTest.java (hunk 1 above)
Comment (DavidRedlich) [reply src/test/java/com/targomo/jackson/datatype/trove/deser/TIntFloatMapDeserializerTest.java]: because that was set for "no_value". (see https://github.com/targomo/targomo-jackson-datatype-trove/blob/bbad1f6b03edde8f0e55a6a89fee86272c216efb/src/test/java/com/targomo/jackson/datatype/trove/deser/BaseTroveDeserializerTest.java#L16)
but perhaps it's better to make this a constant for the tests

Comment (DavidRedlich) [reply src/test/java/com/targomo/jackson/datatype/trove/deser/TIntFloatMapDeserializerTest.java]: have it as a constant in the test now

File: src/test/java/com/targomo/jackson/datatype/trove/deser/TIntFloatMapDeserializerTest.java (hunk 2 above)
Comment (DavidRedlich) [reply src/test/java/com/targomo/jackson/datatype/trove/deser/TIntFloatMapDeserializerTest.java]: uncommented all sysos - so that there is no output in the tests anymore.

File: src/test/java/com/targomo/jackson/datatype/trove/ser/TIntFloatMapSerializerTest.java (hunk 3 above)
Comment (DavidRedlich) [reply src/test/java/com/targomo/jackson/datatype/trove/ser/TIntFloatMapSerializerTest.java:17]: added the edge test as well
